services/vqa-service/service_metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": 1,
+    "display_name": "ProcessService",
+    "encoding": "proto",
+    "service_type": "grpc",
+    "payment_expiration_threshold": 40320,
+    "model_ipfs_hash": "QmTAXBccCXmN2G6fjzBpME8aaBiLdsp8fNTLAD2mbLCjak",
```

Comment:
some of the fields like IPFS hashes and even contract addresses here are subjects to change. If you would like to save the infrastructure to deploy service easily it is better to save it as shell script which executes `snet-cli` to deploy the application and has parameters like endpoints and price.